### ISSUE TITLE
GNOME style instead Unity

### ISSUE THREAD

**Issue by jonaaix** [MauriceConrad/Photon#28]
Wouldn't it make more sense to implement the GNOME stylesheets instead of any kind of Unity UI since there exist detailed guidelines for designing gnome styled applications. 
[https://developer.gnome.org/hig/stable/](https://developer.gnome.org/hig/stable/)
[https://developer.gnome.org/hig/stable/ui-elements.html.de](https://developer.gnome.org/hig/stable/ui-elements.html.de)

**Comment by ElyaConrad** [comment]
Yes, it definitely makes more sense. But at the moment, there exist neither a GNOME stylesheet than a metro ;-) The related style name "unity" is just an example in the README.md and has no dependency in the code. But I would be very happy if a GNOME developer would implement GNOME stylesheets ;-) Currently, I do not have the capacities to do so.
